Sources/Sentry/include/SentryTime.h

```diff
@@ -11,6 +11,12 @@ SENTRY_EXTERN_C_BEGIN
  */
 uint64_t timeIntervalToNanoseconds(double seconds);
 
+/**
+ * Converts nanoseconds to a @c double from a Cocoa API like @c -[NSDate
+ * @c timeIntervalSinceDate:] .
+ */
+double nanosecondsToTimeInterval(uint64_t nanoseconds);
```

Comment:
Let's use actual `NSTimeInterval`, which is a `double` typedef and has its own docs that people can read.

```suggestion
/** Converts integer nanoseconds to a @c NSTimeInterval. */
NSTimeInterval nanosecondsToTimeInterval(uint64_t nanoseconds);
```